Resources.it.resx

```diff
@@ -0,0 +1,1043 @@
+  </data>
+  <data name="SettingsAlertFontColorTooltip" xml:space="preserve">
+    <value>Colore del testo di avviso.</value>
+    <comment>Settings alert font color tooltip</comment>
+  </data>
+  <data name="SettingsAllCoreClocks" xml:space="preserve">
+    <value>Mostra tutti i core clock</value>
+    <comment>Settings all core clocks label</comment>
+  </data>
+  <data name="SettingsAllCoreClocksTooltip" xml:space="preserve">
+    <value>Mostra le velocità di clock di tutti i core, non solo il primo.</value>
+    <comment>Settings all core clocks tooltip</comment>
+  </data>
+  <data name="SettingsAlwaysOnTop" xml:space="preserve">
+    <value>Sempre in primo piano</value>
+    <comment>Settings always on top label</comment>
+  </data>
+  <data name="SettingsAlwaysOnTopTooltip" xml:space="preserve">
+    <value>Rende la barra laterale sempre sopra le altre finestre.</value>
+    <comment>Settings always on top tooltip</comment>
+  </data>
+  <data name="SettingsAutoBackground" xml:space="preserve">
+    <value>Colore di sfondo automatico</value>
+    <comment>Settings auto background label</comment>
+  </data>
+  <data name="SettingsAutoBackgroundTooltip" xml:space="preserve">
+    <value>Imposta il colore di sfondo sulle impostazioni del colore di Windows.</value>
+    <comment>Settings auto background tooltip</comment>
+  </data>
+  <data name="SettingsAutoUpdate" xml:space="preserve">
+    <value>Aggiornamento automatico</value>
+    <comment>Settings auto update label</comment>
+  </data>
+  <data name="SettingsAutoUpdateTooltip" xml:space="preserve">
+    <value>Aggiorna l'app automaticamente.</value>
+    <comment>Settings auto update tooltip</comment>
+  </data>
+  <data name="SettingsBackgroundColor" xml:space="preserve">
+    <value>Colore di sfondo</value>
+    <comment>Settings background color label</comment>
+  </data>
+  <data name="SettingsBackgroundColorTooltip" xml:space="preserve">
+    <value>Colore dello sfondo.</value>
+    <comment>Settings background color tooltip</comment>
+  </data>
+  <data name="SettingsBackgroundOpacity" xml:space="preserve">
+    <value>Opacità di sfondo</value>
+    <comment>Settings background opacity label</comment>
+  </data>
+  <data name="SettingsBackgroundOpacityTooltip" xml:space="preserve">
+    <value>Opacità dello sfondo.</value>
+    <comment>Settings background opacity tooltip</comment>
+  </data>
+  <data name="SettingsBandwidthInAlert" xml:space="preserve">
+    <value>Allert banda in entrata</value>
+    <comment>Settings bandwidth in alert label</comment>
+  </data>
+  <data name="SettingsBandwidthInAlertTooltip" xml:space="preserve">
+    <value>Soglia kbps per avvisi su larghezza di banda in entrata. Usa 0 per disabilitare.</value>
+    <comment>Settings bandwidth in alert tooltip</comment>
+  </data>
+  <data name="SettingsBandwidthOutAlert" xml:space="preserve">
+    <value>Allert banda in uscita</value>
+    <comment>Allert banda in uscita</comment>

```

Comment:
Final change request
Please revert the `<comment>` back to its english text, like shown below:

```suggestion
    <comment>Settings bandwidth out alert label</comment>
```